### ISSUE TITLE
sqlplus does not exit after running build

### ISSUE THREAD

**Issue by mortenbra** [zabel-xyz/plsql-language#5]
The readme suggests making a task to run sqlplus to build/compile code.

Unless the script/file contains an "exit" statement, sqlplus will hang in the background and never complete, as described here: http://serverfault.com/questions/87035/run-oracle-sql-script-and-exit-from-sqlplus-exe-via-command-prompt

A solution that works better is to create a bat file, such as `run_sqlplus.bat` and place the following command inside it:

`echo exit | echo show errors | sqlplus %1 %2`

The build task definition then becomes:

```
 {
        "version": "0.1.0",

        // The command is a shell script
        "isShellCommand": true,

        // Run sqlplus via a batch file
        "command": "run_sqlplus.bat",

        "args": ["username/password@sid", "@\"${file}\""]
    }
```

This will run sqlplus, output any errors, and then exit cleanly back to VS Code.

It would be great if you could update the readme with this workaround/improvement.

**Comment by zabel-xyz** [comment]
Thank you for sharing this information. I'll include it in the readme...

**Comment by zabel-xyz** [comment]
fix in version 0.0.9

**Comment by APerCova** [comment]
Another approach can be to add an "echo" command with the "sqlplus" command as a command argument as shown:
```javascript
"tasks":[
        //Run sqlplus in args
        {
        "taskName":"sqlplus",
        "command":"echo", 
        "args":["exit","|","echo","show","errors","|","sqlplus","username/password@sid","@\"${file}\""],
        "isShellCommand": true,
        "showOutput": "always"
        }
    ]
```
